appmetrics/app.go

```diff
@@ -104,32 +126,13 @@ func (a *App) mkMetrics(names []string, ms []float64, moreTags []string) []metri
 	return metricsPackages
 }
 
-func (a *App) parseContainerMetric(message *events.ContainerMetric) ([]metrics.MetricPackage, error) {
-	var names = []string{
-		"app.cpu.pct",
-		"app.disk.used",
-		"app.disk.quota",
-		"app.memory.used",
-		"app.memory.quota",
-	}
-	var ms = []float64{
-		float64(message.GetCpuPercentage()),
-		float64(message.GetDiskBytes()),
-		float64(message.GetDiskBytesQuota()),
-		float64(message.GetMemoryBytes()),
-		float64(message.GetMemoryBytesQuota()),
-	}
-	tags := []string{fmt.Sprintf("instance:%v", message.GetInstanceIndex())}
-
-	return a.mkMetrics(names, ms, tags), nil
-}
-
 func (a *App) getTags() []string {
 	if a.Tags != nil && len(a.Tags) > 0 {
 		return a.Tags
 	}
 
-	return a.generateTags()
+	a.Tags = a.generateTags()
+	return a.Tags
 }
```

Comment:
Note: changed this because `a.generateTags()` was returning the generated tags _as well as_ setting the app's tags as a side effect, and that wasn't clear